apps/src/templates/feedback/GeneratedCode.jsx

```diff
@@ -12,17 +13,14 @@ export default class GeneratedCode extends React.Component {
   render() {
     return (
       <div className="generated-code-container" style={this.props.style}>
-        <p
-          className="generatedCodeMessage"
-          dangerouslySetInnerHTML={{__html: this.props.message}}
-        />
+        <div className="generatedCodeMessage">
+          <UnsafeRenderedMarkdown markdown={this.props.message} />
+        </div>
 
         {/* code container should be LTR even in RTL mode */}
-        <pre
-          className="generatedCode"
-          dir="ltr"
-          dangerouslySetInnerHTML={{__html: this.props.code}}
-        />
+        <pre className="generatedCode" dir="ltr">
+          {this.props.code}
+        </pre>
```

Comment:
I take it this `dangerouslySetInnerHTML` was never needed for `props.code`?

Reply:
As far as I can tell, that's correct! The content in here comes from `codegen`, which never uses html characters.

Which also makes sense; there's no reason to `dangerouslySetInnerHTML` on a `pre` tag, since it would just be ignored anyway.